src/ui/popup/settings-page/devtools.tsx

```diff
@@ -5,14 +5,55 @@ import {Button} from '../../controls';
 import ControlGroup from '../control-group';
 import {ViewProps} from '../types';
 
-function openDevTools() {
-    // TODO: Switch to already opened window.
-    chrome.windows.create({
-        type: 'panel',
-        url: 'ui/devtools/index.html',
-        width: 600,
-        height: 600,
-    });
+
+function getExistingDevToolsWindowId() {
```

Comment:
Replace `getExistingDevToolsWindowId` and `isDevToolsOpen` with one `getExistingDevToolsWindow` function. Assign a return type as `Promise<chrome.Window>` or other available type.